src/components/PersonaWrap/PersonaWrap.scss

```diff
@@ -102,21 +89,21 @@ $block: '.#{variables.$ns}persona';
             margin-right: 6px;
         }
 
-        #{$block}_clickable:hover & {
-            background-color: var(--g-color-base-generic);
-        }
+        #{$blockAvatar} {
+            background-color: var(--g-color-base-generic-accent);
 
-        span {
-            @include mixins.text-accent;
-            text-transform: uppercase;
-            font-size: 11px;
-            line-height: 11px;
-        }
+            transition-property: background-color;
+            transition-duration: $transitionDuration;
+            transition-timing-function: $transitionTimingFunction;
 
-        img {
-            width: $avatarSize;
-            height: $avatarSize;
-            object-fit: cover;
+            & #{$blockAvatar}__icon,
+            & #{$blockAvatar}__text {
```

Comment:
This should be done with CSS API (https://github.com/gravity-ui/uikit/blob/6805305b58ebee3f7d9242170641f87ff48aad61/src/components/Button/README.md#css-api), without using classes